src/components/Preferences/ModalActions.vue

```diff
@@ -24,16 +55,26 @@ export default Vue.extend({
         return 'restart';
       }
 
+      if (this.severities.error) {
+        return 'error';
+      }
+
       return '';
     },
-    severityLevel() {
-      return this.severity === 'reset' ? 'warning' : 'info';
+    alert(): Alert {
+      return alertMap[this.severity];
     },
-    iconClass() {
-      return `icon-${ this.severityLevel }`;
+    errorFormatted(): string {
+      return `${ this.preferencesError.charAt(0).toUpperCase() }${ this.preferencesError.slice(1) }`;
```

Comment:
Hmm, munging the text like this is icky.

But I'm not sure a combination of [`::first-letter`](https://developer.mozilla.org/en-US/docs/Web/CSS/::first-letter) + [`text-transform: capitalize`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) is any better; ideally we'd have properly localized messages instead…

Reply:
I agree with the ickiness. I ran into some roadblocks when using `::first-letter` that could be related to lack of specificity.. Instead of spending more time troubleshooting, I opted to just drop the formatting in the renderer and fix it at the source in `background.ts`